src/components/bookmark-button/bookmark-button.module.css

```diff
@@ -3,16 +3,17 @@
 	--hoverIconOpacity: 0;
 	--baseIconOpacity: 1;
 
-	color: var(--token-color-foreground-faint);
-	position: relative;
-	display: flex;
 	align-items: center;
-	cursor: pointer;
 	background: none;
 	border: none;
+	color: var(--token-color-foreground-faint);
+	cursor: pointer;
+	display: flex;
+	height: var(--buttonSize);
 	padding: 0;
+	position: relative;
 	width: var(--buttonSize);
-	height: var(--buttonSize);
+	z-index: 1;
```

Comment:
🗒️ This was needed to place the button on top of `CardLink`'s `a::before` element.